src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -48,6 +48,7 @@ export type PackageUpdateOptions = {
   Name?: string;
   Description?: string;
   PackageErrorUsername?: string;
+  AppAnalyticsEnabled?: boolean;
```

Comment:
@stephen-carter-at-sf this property should also be added to the Package2 type [here](https://github.com/forcedotcom/packaging/blob/main/src/interfaces/packagingSObjects.ts#L11).